### ISSUE TITLE
Export of SOP geometry from a RayTK scene

### ISSUE THREAD

**Issue by t3kt** [t3kt/raytk#794]

**Comment by Empyreans** [comment]
If I am not mistaken, Houdini for example should be able to convert said points to a mesh and has a flexible toolkit available for experimenting. Is it possible to export these points from raytk as of yet?

**Comment by t3kt** [comment]
Yeah Houdini can do it. To export the points, you'd use a pointMapRender to sample the SDF at points in a grid, then use the resulting distance values to delete outside points (those with positive distances).

I've had some success with using the [Convex Hull SOP](https://github.com/colasf/convexhullsop) plugin to do that conversion. It only works for convex shapes though, sort of like shrink-wrapping a shape without getting all the inner details.

I've also successfully done the reverse in Houdini, taking a model and producing SDF data out of it (with a lot of limitations). See #657 for more about that.
